current_range_test.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from unittest.mock import patch
+from io import StringIO
+import sys
+from current_range import detect_and_display_current_readings
+
+STD_OUT = 'sys.stdout'
+
+class CurrentRangeTest(unittest.TestCase):
```

Comment:
Good start !
Test cases need to be strengthened with single valued list, multi valued list with repetitions, non consecutive values etc.
Please add more test cases.